### ISSUE TITLE
[TFT] Missing a code to apply the optimal learning rate found

### ISSUE THREAD

**Issue by jiryang** [jdb78/pytorch-forecasting#298]
- PyTorch-Forecasting version: 0.8.2
- PyTorch version: 1.8.0
- Python version: 3.6.9
- Operating System: Ubuntu 18.04

Nothing major but, a line to apply the optimal learning rate to the model in the TFT tutorial seems to be omitted.

Maybe the below line after [8]:
tft.hparams.learning_rate = res.suggestion()

**Comment by jdb78** [comment]
It is rather dangerous to directly take the suggested learning rate without visual inspection. I do not recommend it because sometimes the "suggestion" is pretty wrong but it will be obvious from visual inspection.
